photon-api/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimator.scala

```diff
@@ -507,12 +508,32 @@ class GameEstimator(val sc: SparkContext, implicit val logger: Logger) extends P
             None
           }
 
-          val randomEffectDataset = RandomEffectDataset(
-            gameDataset,
-            reConfig,
-            rePartitioner,
-            existingModelKeysRddOpt,
-            StorageLevel.DISK_ONLY)
+          val randomEffectDataset = if(initialModelOpt.isDefined) {
+            val reModels = initialModelOpt.get.getModel(coordinateId).map {
+              case rem: RandomEffectModel =>
```

Comment:
The parameter `rem` is not self-explanatory. Maybe use `reModel`?

Reply:
fixed